debug/src/debug.js

```diff
@@ -376,7 +376,7 @@ Component.prototype.forceUpdate = function(callback) {
 		);
 	} else if (this._parentDom == null) {
 		console.warn(
-			`Can't call "this.setState" on an unmounted component. This is a no-op, ` +
+			`Can't call "this.forceUpdate" on an unmounted component. This is a no-op, ` +
```

Comment:
Wondering if we should mention both here. The user is probably not aware that we call `forceUpdate` for `setState` under the hood.

Reply:
I think since #1939 forceUpdate and setState and fully independent if I'm reading this correctly.